flair/nn/model.py

```diff
@@ -737,7 +743,8 @@ def _encode_data_points(self, sentences: List[DT], data_points: List[DT2]):
 
         return data_point_tensor
 
-    def _mask_scores(self, scores, data_points):
+    def _mask_scores(self, scores: Tensor, data_points) -> Tensor:
+        """This appears to have not been fully implemented and simply returns scores."""
```

Comment:
In most models, scores are not masked, so the default implementation is to just return the scores. The `SpanClassifier` overwrites this method to account for "candidate lists". Anything not on these lists, will be masked.